kahenya-anita/Bumble-Clone
# Iframe anita kahenya

## Description
Changed the folder structure to use the root files for HTML & CSS.
## Ticket [Iframe](https://sharing.clickup.com/36234712/t/h/8669fjb62/S8VJQ6OEMXKW405)
## Types Of Changes
[] Bug fix

[] New Feature

[X] Breaking Changes

## Checklist:
[X]My code follows the code style of this project

[X]All new and existing tests passed.

[X]My code has been linted and formatted correctly.

[x]I have commented my code, particularly in hard-to-understand areas.

[X]My changes generate no new warnings.

## How can this code be tested?
1. After Cloning this repo , run `git pull
` to ensure you are up to date. 
2. Checkout to [Iframe_Anita-Kahenya](Iframe_Anita-Kahenya)

<img width="1440" alt="Screenshot 2023-03-12 at 7 51 40 PM" src="https://user-images.githubusercontent.com/62019551/224559723-f3ab5b64-b8f8-4214-97c9-91f1cb516443.png">
![Screenshot 2023-03-12 at 7 51 40 PM (2)](https://user-images.githubusercontent.com/62019551/224559729-442eb069-2b74-4ad3-823c-954041508a2e.png)

4. Run the changes in the browser
![Screenshot 2023-03-12 at 7 52 57 PM](https://user-images.githubusercontent.com/62019551/224559783-5e652192-4a8f-4667-b85c-1070024d08e4.png)
